.travis.yml

```
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "7"
+  - "8"
+dist: trusty
+env:
+cache:
+  yarn: true
+  directories:
+    - node_modules
+install:
+  - yarn install
+  # puppeteer's install script downloads Chrome
+script:
+  - yarn run lint
```

these 3 are in `yarn test`

---

agree. :)

they're separate here (and in LH's) so that the entire chain doesn't fail and quit if there's an error in the first command.

basically, if there's a lint error, it's still useful to let the remaining tests run.

---

Since PR merges get blocked until everything passes, it's kinda nice to fail fast. Travis is known to take forever. But nbd.

---

yes. i wish it were possible to flag the build as failing immediately while letting the rest finish.

but it's not. we looked into it a few weeks ago on LH.